src/sys/wait.rs

```diff
@@ -112,149 +96,52 @@ impl WaitStatus {
     }
 }
 
-#[cfg(any(target_os = "linux",
-          target_os = "android"))]
 mod status {
     use sys::signal::Signal;
-    use libc::c_int;
-    use libc::SIGTRAP;
+    use libc::{self, c_int, SIGTRAP};
 
     pub fn exited(status: i32) -> bool {
-        (status & 0x7F) == 0
+        unsafe { libc::WIFEXITED(status) }
```

Comment:
These are defined as macros or functions by the spec, so it makes sense to keep them in `libc` even if they're macros on some platforms and need to be implemented.